tox.ini

```diff
@@ -12,6 +12,8 @@ deps =
     coverage
     scikit-learn
     python-Levenshtein
+    moviepy
+    sndfile 
```

Comment:
These two (moviepy and sndfile) can be removed here. If they are in setup.py, they will automatically be installed via pip in the tox run.

Reply:
Actually, I think scikit-learn and python-Levenshtein should also be removed here, they should also be installed through pip via setup.py. Could you try that?